### DISCUSSION TITLE
I cannot change themes.

### DISCUSSION THREAD

**Discussion by JNewstedt** [OrchardCMS/OrchardCore#16804] [Q&A]
I am new to Orchard Core and the site's docs aren't very clear, to say the least. However, I have managed to get as far as creating a site in Visual Studio. My problem is that I cannot change my site's theme after selecting "Make Current" in Design > Themes. I still get the stock web theme. Am I missing a setting or something?

**Comment by Piedone** [comment]
So on the /Admin/Theme page, you click a button like this, then go back to the frontend, and nothing changes?

![image](https://github.com/user-attachments/assets/3a64d411-4b23-4b7a-904e-8248d345d803)

Have you seen this too? https://docs.orchardcore.net/en/latest/resources/tutorials/

**Reply by JNewstedt** [reply]
Yes, nothing changes. I can get to the Themes page without a problem, but I still see the same .NET-generated white page and text. 

I have also been through the docs on the Orchard site, but have noticed the wording is confusing at times and some of the info is outdated, still referring to the Startup.cs page (which has been discontinued). 

I'm waiting for more current tutorials to come out that have clear step-by-step instructions for setting up Orchard sites. I haven't had much luck finding newer tutorials; they're usually around 3-4 years old.

**Reply by Piedone** [reply]
Please provide a screenshot of the white page and text.

**Reply by JNewstedt** [reply]
It looks like this:

![test](https://github.com/user-attachments/assets/fc8dc3af-c48e-4bb0-9d17-b6aa0996fdcf)


I should note that the admin page looks the same as your example. But when I choose a recipe and click "Make Current," the homepage stays the same.

**Reply by Piedone** [reply]
This is indeed the default ASP.NET Core homepage, and not the themed Orchard Core admin. So, you perhaps did something differently than in the tutorials.

Anyway, I recommend just [installing the Lombiq Utility Scripts](https://github.com/Lombiq/Utility-Scripts?tab=readme-ov-file#installing-the-powershell-modules), then running `Initialize-OrchardCoreSolution -Name "MyWebsite"` to generate the solution with everything, and be done with it.

**Reply by JNewstedt** [reply]
I don't know what happened, but things appear to work after the latest .NET update.

**Reply by Piedone** [reply]
OK then!
